src/instructlab/server.py

```diff
@@ -163,6 +176,16 @@ def server(
     queue=None,
 ):
     """Start OpenAI-compatible server"""
+    if not is_valid_model_path(logger, model_path):
+        sys.stdout = sys.__stdout__  # Reset standard output to its default
+        print(
+            "---\nPlease verify the model path in 'config.yaml' or '--model-path'. Ensure it is readable and has a '.gguf' extension."
```

Comment:
Would drop ggug extension text from here as well. Maybe replace with 

s/has a '.gguf' extension/is in a model file format

?

Reply:
Sure I can remove the logic for checking specifically for gguf.